docker-compose.yml

```diff
@@ -71,6 +71,7 @@ balrogui:
     - .cache:/cache
   environment:
     - WEB_PORT=8080
+    - LOG_LEVEL=WARNING
```

Comment:
I don't think this will do anything for the ui and db containers. Remove this and then it's good to go!